packages/editor-ui/src/Interface.ts

```diff
@@ -314,8 +314,29 @@ export interface IWorkflowDb {
 	versionId: string;
 	usedCredentials?: IUsedCredential[];
 	meta?: WorkflowMetadata;
+	resource?: 'workflow';
 }
 
+// For workflow list we don't need the full workflow data
+export type WorkflowResourceDB = Omit<
```

Comment:
maybe we can drop the `DB` postfix here, not sure it's adding anything.

Reply:
Makes sense. I wanted to keep it in line with our other naming (like `IWorkflowDB`) but will updated